apis/nucleus/src/components/listbox/default-properties.js

```diff
@@ -1,10 +1,35 @@
+/**
+ * see: https://qlik.dev/apis/json-rpc/qix/schemas#%23%2Fdefinitions%2Fschemas%2Fentries%2FValueExpression
+ * @name ValueExpression
+ * @type object
+ * @property {string} qValueExpression.qExpr
+ */
+
+/**
+ * @name ListboxProperties
+ * @type object
+ */
+
+/**
+ * @lends ListboxProperties
+ */
 const listdef = {
   qInfo: {
     qType: 'njsListbox',
   },
+  /**
+   * Extends `ListObjectDef`, see Engine API: `ListObjectDef`.
+   * @extends {EngineAPI.IListObjectDef}
+   */
```

Comment:
This looks to give problem with the generate typescript types step.
unsure what to use instead or if it should be fixed in `@scriptappy/to-dts`

Reply:
Could it be that `qListObjectDef` needs to be defined as an interface in order to extend `EngineAPI.IListObjectDef`?

Reply:
I did not get that to work correctly.
(but I got a version that ignored `@extends` instead of crashing)

Reply:
Have you tried without the curly brackets?

Reply:
I get different problem (but still not working)